src/main/kotlin/io/github/inductiveautomation/kindling/log/LogDetailsPane.kt

```diff
@@ -0,0 +1,245 @@
+                    }
+                    if (event.message != null) {
+                        append("<b>Message:</b><br>")
+                        append(event.message.escapeHtml().replace("\t", "       "))
+                        append("<br>")
+                    }
+                    if (event.body.isNotEmpty()) {
+                        append("<br><b>StackTrace:</b><br>")
+                        event.body.joinTo(buffer = this, separator = "<br>") { (text, link) ->
+                            if (link != null) {
+                                """<a href="$link">$text</a>"""
+                            } else {
+                                text.replace("\t", "       ")
+                            }
+                        }
+                    }
+                }
+            }
+        }
+
+        init {
+            border = BorderFactory.createLineBorder(UIManager.getColor("Component.borderColor"), 1)
+            add(header, "growx, wrap")
+            add(collapsablePane, "growx")
+        }
+    }
+
+    private fun List<Detail>.toClipboardFormat(): String {
+        return joinToString(separator = "\n\n") { event ->
+            buildString {
+                appendLine(event.title)
+                if (event.message != null) {
+                    appendLine(event.message)
+                }
+                event.body.joinTo(buffer = this, separator = "\n") { "\t${it.text}" }
+            }
+        }
+    }
+
+    private fun List<Detail>.toPopoutFormat(): String {
+        return joinToString(separator = "", prefix = "<html><pre>") { event ->
+            buildString {
+                append("<b>${event.title.escapeHtml()}</b><br>")
+                if (event.details.isNotEmpty()) {
+                    event.details.entries.joinTo(buffer = this, separator = " ") { (key, value) ->
+                        "data-$key = \"$value\""
+                    }
+                    append("<br>")
+                }
+                if (event.message != null) {
+                    append(event.message.escapeHtml().replace("\t", "       "))
+                    append("<br>")
+                }
+                if (event.body.isNotEmpty()) {
+                    event.body.joinTo(buffer = this, separator = "<br>") { (text, link) ->
+                        if (link != null) {
+                            """<a href="$link">$text</a>"""
+                        } else {
+                            text.replace("\t", "       ")
+                        }
+                    }
+                    append("<br>")
+                }
+            }
+        }
+    }
+}
+
+
+class DetailsEditorKit : HTMLEditorKit() {
+    init {
+        styleSheet.apply {
+            //language=CSS
+            addRule(
+                    """
+                b { 
+                    font-size: larger; 
+                }
+                pre { 
+                    font-size: 10px; 
+                    font-family: Segoe
```

Comment:
I think the justification was "Dialog is ugly" but we can probably just use it

Reply:
I believe I originally made this switch because the log details had two different fonts, one for the stacktrace and one for the message. This was an attempt to make it consistent, however this issue is accounted for in other ways now that the whole details pane is restructured.

Reply:
I'm open to including a FOSS font like one of the ones from FlatLaf if that's preferred:
https://github.com/JFormDesigner/FlatLaf/tree/main/flatlaf-fonts